src/Particles/babylon.particle.ts

```diff
@@ -10,6 +11,28 @@
         public angle = 0;
         public angularSpeed = 0;
 
+        constructor(private particleSystem: ParticleSystem, public cellWidth: number = 0, public cellHeight: number = 0, public cellIndex: number = 0, public invertU: number = 0, public invertV: number = 0, private _loopAnimation = false, private _fromIndex = 0, private _toIndex = 0, private _delay = 0, private _sheetDirection = 1, private _time = 0, private disposeWhenFinishedAnimating = false) {
+        }
+
+        public _animate(deltaTime: number): void {

```

Comment:
I'm wondering why this should not be just part of the particle update process? It is not different from position animation for instance and we should not treat the cellIndex differently that the position, color or all other particle properties